sources/metrics/open measurement/OpenMeasurementContext.swift

```diff
@@ -16,13 +16,17 @@ enum OpenMeasurement {
         let adEvents: PlayerCore.OpenMeasurement.AdEvents
         let videoEvents: PlayerCore.OpenMeasurement.VideoEvents
     }
-    enum Errors: Swift.Error {
+    enum Errors: CustomNSError {
         case failedToCreateOMIDPartner
         case failedToCreateVerificationScriptResource
         case failedToActivateSDK
         case scriptNotAvailable
         case failedToGetAdView
         case sdkVersionIsNotCompatible
+        
+        var errorUserInfo: [String : Any] {
```

Comment:
just `errors` should be enough - and where it is used?

Reply:
No, it's not enough. In case of just error we will see `Swift.Error(error with code: 3)` on the reporting side. But If we use `CustomNSError` we will see `failedToCreateOMIDPartner` or some other human readable error.

Reply:
ok, got it